### PR TITLE
Fixed issue in qf round project cap limit

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

File: src/resolvers/roundsResolver.test.ts [hunk 1]
```diff
@@ -59,17 +59,17 @@ function fetchAllRoundsTestCases() {
       roundNumber: generateEARoundNumber(),
       startDate: new Date(),
       endDate: moment().add(3, 'days').toDate(),
-      roundUSDCapPerProject: 1000000,
-      roundUSDCapPerUserPerProject: 50000,
+      roundUSDCapPerProject: 1_000_000,
```
Comment (lovelgeorge99) [inline src/resolvers/roundsResolver.test.ts:62]: why did you change it from 1000000 to 1_000_000 ?

Comment (aminlatifi) [reply src/resolvers/roundsResolver.test.ts:62]: It's just a js notation for redability

**Review by lovelgeorge99 — APPROVED** [review]
LGTM 👍
